src/components/ChannelGrid.svelte

```diff
@@ -27,9 +28,9 @@
           </div>
         </div>
         <div class="bg-white dark:bg-gray-800">
-          {#each channels as channel (channel.id)}
-            <ChannelItem bind:channel />
-          {/each}
+          <VirtualList height="80vh" items={channels} let:item>
```

Comment:
```suggestion
          <VirtualList height="60vh" items={channels} let:item>
```
Since the scrollbar exists inside why not remove the outer scroll?